x/valset/keeper/grpc_query_get_alive_pigeons.go

```diff
@@ -21,10 +22,15 @@ func (k Keeper) GetAlivePigeons(goCtx context.Context, req *types.QueryGetAliveP
 	vals := k.GetUnjailedValidators(ctx)
 
 	res := slice.Map(vals, func(val stakingtypes.ValidatorI) *types.QueryGetAlivePigeonsResponse_ValidatorAlive {
-		until, err := k.ValidatorAliveUntil(ctx, val.GetOperator())
+		bz, err := keeperutil.ValAddressFromBech32(k.AddressCodec, val.GetOperator())
+
 		s := &types.QueryGetAlivePigeonsResponse_ValidatorAlive{
-			ValAddress: val.GetOperator(),
+			ValAddress: bz,
+		}
+		if err != nil {
+			s.Error = err.Error()
```

Comment:
Move error handling to after assignment, and what should happen in case we actually encounter an error here? probably we do not just want to continue with regular logic flow?